l10n_jp_country_state/readme/USAGE.rst

```diff
@@ -0,0 +1 @@
+This module will load translation of Japanese prefectures.
```

Comment:
Sorry it's a minor point but can you please override DESCRIPTION.rst with this content and then delete this file, since this module no longer updates the country state data?

Reply:
Thank you.
I removed USAGE.rst, and overwrote DESCRIPTION.rst.
https://github.com/OCA/l10n-japan/pull/20/commits/916ec88c83acd128c9c3388669d434ae6865e8ea#diff-bbc2191399167eb5bd1afaadc95c08270f0af3eda833d49b08a187cc054b5a4cL1-R2